lib/Event.php

```diff
@@ -27,121 +28,121 @@ class Event extends ApiResource
-    const ORDER_UPDATED                        = 'order.updated';
-    const ORDER_RETURN_CREATED                 = 'order_return.created';
-    const PAYOUT_CANCELED                      = 'payout.canceled';
-    const PAYOUT_CREATED                       = 'payout.created';
-    const PAYOUT_FAILED                        = 'payout.failed';
-    const PAYOUT_PAID                          = 'payout.paid';
-    const PAYOUT_UPDATED                       = 'payout.updated';
-    const PING                                 = 'ping';
-    const PLAN_CREATED                         = 'plan.created';
-    const PLAN_DELETED                         = 'plan.deleted';
-    const PLAN_UPDATED                         = 'plan.updated';
-    const PRODUCT_CREATED                      = 'product.created';
-    const PRODUCT_DELETED                      = 'product.deleted';
-    const PRODUCT_UPDATED                      = 'product.updated';
-    const RECIPIENT_CREATED                    = 'recipient.created';
-    const RECIPIENT_DELETED                    = 'recipient.deleted';
-    const RECIPIENT_UPDATED                    = 'recipient.updated';
-    const REPORTING_REPORT_RUN_FAILED          = 'reporting.report_run.failed';
-    const REPORTING_REPORT_RUN_SUCCEEDED       = 'reporting.report_run.succeeded';
-    const REPORTING_REPORT_TYPE_UPDATED        = 'reporting.report_type.updated';
-    const REVIEW_CLOSED                        = 'review.closed';
-    const REVIEW_OPENED                        = 'review.opened';
-    const SIGMA_SCHEDULED_QUERY_RUN_CREATED    = 'sigma.scheduled_query_run.created';
-    const SKU_CREATED                          = 'sku.created';
-    const SKU_DELETED                          = 'sku.deleted';
-    const SKU_UPDATED                          = 'sku.updated';
-    const SOURCE_CANCELED                      = 'source.canceled';
-    const SOURCE_CHARGEABLE                    = 'source.chargeable';
-    const SOURCE_FAILED                        = 'source.failed';
-    const SOURCE_MANDATE_NOTIFICATION          = 'source.mandate_notification';
-    const SOURCE_REFUND_ATTRIBUTES_REQUIRED    = 'source.refund_attributes_required';
-    const SOURCE_TRANSACTION_CREATED           = 'source.transaction.created';
-    const SOURCE_TRANSACTION_UPDATED           = 'source.transaction.updated';
-    const SUBSCRIPTION_SCHEDULE_ABORTED        = 'subscription_schedule.aborted';
-    const SUBSCRIPTION_SCHEDULE_CANCELED       = 'subscription_schedule.canceled';
-    const SUBSCRIPTION_SCHEDULE_COMPLETED      = 'subscription_schedule.completed';
-    const SUBSCRIPTION_SCHEDULE_CREATED        = 'subscription_schedule.created';
-    const SUBSCRIPTION_SCHEDULE_EXPIRING       = 'subscription_schedule.expiring';
-    const SUBSCRIPTION_SCHEDULE_RELEASED       = 'subscription_schedule.released';
-    const SUBSCRIPTION_SCHEDULE_UPDATED        = 'subscription_schedule.updated';
```

Comment:
Please don't remove those. It'd be a breaking change. They will be released in the future.